actions.yaml

```diff
@@ -0,0 +1,2 @@
+report-visits-by-ip:
+  description: Look at the proxy log and list the IPs that visited the proxy the most.
```

Comment:
Only from the last 2 hours, right?

Reply:
Taking another look at the bug, it should be in the last 20 minutes, so I'll modify both the action and the description